src/api/account_api.js

```diff
@@ -620,6 +620,7 @@ module.exports = {
             type: 'object',
             required: ['name', 'email', 'has_s3_access'],
             properties: {
+                _id: { type: 'string'},
```

Comment:
I think it would be confusing that in containerized the `account_info` will have `_id` as string.
I thought maybe to change the property name (`account_id`), but I'm not sure.

Reply:
I thought is might be confusing because `_id` is usually for `ObjectId`.
So in other places in the code to see an ID as string we should have `account._id.tostring()`.